### PR TITLE
Use resin uboot bbclass

### PR DESCRIPTION
**Pull request by floion**

### CODE REVIEW

**Review by agherzan — APPROVED** [review]
Please merge with a released version of meta-resin (not an intermediate revision). So we can have a known version in resin-board.

**Comment by floion** [comment]
There is no released version of meta-resin which has the 2.0 uboot spec as far as I recall.

**Comment by agherzan** [comment]
I'm just saying that would be better to wait for a release and incorporate it in this PR before merging.

**Review by michal-mazurek — APPROVED** [review]

**Pull request merged**
